api/config/config.go

```diff
@@ -280,6 +281,16 @@ type StandardTransformerConfig struct {
 	DefaultFeastSource spec.ServingSource `envconfig:"DEFAULT_FEAST_SOURCE" default:"BIGTABLE"`
 	Jaeger             JaegerConfig
 	SimulationFeast    SimulationFeastConfig
+	Kafka              KafkaConfig
+}
+
+// Kafka configuration
```

Comment:
nit: Make the comment more specific `Kafka configuration for publishing prediction log`